test/integration/concrete/lockup-linear/streamed-amount-of/streamedAmountOf.t.sol

```diff
@@ -17,7 +19,28 @@ contract StreamedAmountOf_LockupLinear_Integration_Concrete_Test is
         StreamedAmountOf_Integration_Concrete_Test.setUp();
     }
 
-    function test_StreamedAmountOf_CliffTimeInThePast()
+    modifier givenStatusPendind() {
```

Comment:
Typo (should be "pendin**g**")

Reply:
Thanks for discovering it @IaroslavMazur. @andreivladbrg please feel free to push a commit directly onto staging branch to fix this.

Reply:
e8b9041d79153b00406843ff4a1461868f5507d3